spk/domoticz/Makefile

```diff
@@ -12,7 +12,7 @@ DESCRIPTION_FRE = Domoticz est un système domotique qui vous permet de surveill
 ADMIN_PORT = 8084
 RELOAD_UI = yes
 DISPLAY_NAME = Domoticz
-CHANGELOG = "update to svn revision 2112"
+CHANGELOG = "update to svn revision 2276"
```

Comment:
Yeah, thought I maybe should have that, but technically Domoticz had OpenZWave support, so we haven't added it, we've enabled it. But happy to append that to the message.